crates/ruma-client-api/src/backup.rs

```diff
@@ -135,11 +139,14 @@ pub struct EncryptedSessionDataInit {
 
     /// First 8 bytes of MAC key, encoded in base64.
     pub mac: Base64,
+
+    /// MAC of the key, encoded in base64
```

Comment:
This type should not be changed, the point of this type is to have a stable interface and constructor if new fields to the non-init type are added, the docs on the type tell you this as well:

> /// This struct will not be updated even if additional fields are added to [`EncryptedSessionData`]
/// in a new (non-breaking) release of the Matrix specification.

Reply:
Ah, I missed the comment.  So, do I create a new `...Init` struct for creating v2 data?